packages/avatars/stories/examples/ChromeUsage.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { Chrome, Body, Header, HeaderItem, HeaderItemIcon } from '@zendeskgarden/react-chrome';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import GridIcon from '@zendeskgarden/svg-icons/src/16/grid-2x2-stroke.svg';
+
+export const ChromeUsage: Story = () => {
+  const CustomHeaderItem = HeaderItem as any;
+
+  return (
+    <Chrome isFluid style={{ height: 'auto' }}>
+      <Body>
+        <Header>
+          <CustomHeaderItem as="button">
+            <HeaderItemIcon>
+              <GridIcon />
+            </HeaderItemIcon>
+          </CustomHeaderItem>
+          <CustomHeaderItem as="button" isRound>
+            <Avatar size="extrasmall">
```

Comment:
Why is this Avatar being styled as `isSystem` square? It needs to be round for this combo example.

Reply:
This is randomly happening based depending on styled-components specificity. Going to keep digging into this one.

Reply:
yeah, still weird on https://5f62659c6de450374ab825a5--zendeskgarden-react-components.netlify.app/storybook/?path=/story/components-avatars--chrome-usage

Reply:
I've increased Avatar styling specificity to beat the `img` styles supplied by `react-chrome`